EXPO/src/utils/utility.js

```diff
@@ -0,0 +1,21 @@
+import { Platform, Alert } from "react-native";
+import { STRING_CONSTANTS } from "./string-constant";
+
+export default class Utility {
```

Comment:
what is this class for?Lets not name any class or path as plain utility.js

Reply:
This class for common methods like date,time operations,alerts and null checks.

Reply:
lets not club them all together.The class will inflate and SRP(Single Responsibility Principle) will be broken

Reply:
for eg - break this class down into a another class like
```
export default class Utility {
static showAlertWithMessage(title,message,navigation,onOkPressed,onCancelPressed){
Alert.alert(
      title,
      message,
      [
        {
          text: STRING_CONSTANTS.OK_MSG,
          onPress: () => onOkPressed()
        }
      ],
      { cancelable: false }
    );
}

}
```

Reply:
Utility should act as factory

Reply:
then lets name such classes as `xxxFactory`.It should return a singleton instance of the class being created though